installer/templates/phx_web/router.ex

```diff
@@ -1,10 +1,14 @@
 defmodule <%= web_namespace %>.Router do
-  use <%= web_namespace %>, :router<%= if html do %>
+  use <%= web_namespace %>, :router<%= if html do %><%= if live do %>
+
+  import Phoenix.LiveView.Router<% end %>
```

Comment:
We don't need this here. :) We've already added it to `use MyAppWeb, :router`.

Reply:
I was thinking I prefer it in the router since you don't often create new routers, and when you do, they will often carry their own concerns, ie API only. I don't have a super strong opinion and the import won't hurt other routes. Thoughts?

Reply:
I see your point. Having everything in there feels more consistent thought :/

Reply:
Agree. app_web.ex it is!